src/components/Text/src/Text.vue

```diff
@@ -169,7 +203,9 @@ export default {
 </script>
 
 <style module="$s">
-.Paragraph {
+.Text {
+	margin: 0;
```

Comment:
This will have unintended consequences. I'd recommend changing the classes so the margin: 0 only happens with the headline, title (and maybe label) variants, but regular text has had margins and this would remove a lot of default spacing on those elements.

Reply:
i think it only impacts texts rendered with a `p` element which inherited some margin from the default browser stylesheet. hmmm, i'll chat about this with Lauren and Bashu and see what they say

Reply:
`p` elements already are styled with `margin: 0` in Website's CSS Reset ([github link](https://github.com/squareup/ecom-website/blob/text-to-mtext-conversion/resources/assets/js/prime/src/css/reset.css#L15-L22)). This should be redundant for `p` elements and therefore fine. 

My only concern is if somebody's adding `margin` to their `p` tag through a class, and then we convert it into an MText and override it- but that's mostly happening via `<style scoped>` tags at the Vue component level, and therefore the specificity will override this one, so this works as a default.

Here's an example of the specificity I just discussed. I'm using a `p` tag I converted to MText in EventStatus ([github link](https://github.com/squareup/ecom-website/pull/12057/files#diff-ab9e31e774b1d7332b68b373e05b54b801918d6d570760c698b208ee83bc0a32R55)), using a mock class with the same specificity as this `Text` class we're looking at:
![image](https://user-images.githubusercontent.com/27756206/161307744-b591ba40-c4c7-49e7-878f-8ae496039bff.png)

Reply:
I responded in an different thread but I think it was resolved/collapsed. +1 to Bashu's explanation - `p` tags are currently reset anyway. This just prevents engineers from defining global styles that apply margin, but it doesn't prevent margin overrides by scoped classes. I think that's fine - the whole reason we have so many global styles in website is because it was the only way to share styles between different components, a problem Maker and Maker Theming is meant to directly address.

Reply:
All this context is helping to establish that the impact of removing margins from the the text component would have minimum to no impact for our largest consumer right now. It would still be considered a breaking change based on the code we've already shipped, I still think we could avoid that right now until we can provide a clearer direction or ability to change it.

Some additional discussion and perhaps some documentation around our philosophy about how styling should be done globally when using Maker. For example, we can be opinionated about a reset vs normalize solution, but we should document that and communicate how to manage that output more clearly.

Tailwind takes the reset approach and then communicates where and how it's happening: https://tailwindcss.com/docs/preflight

Bootstrap starts with Normalize and then makes more opinionated targets styles: https://getbootstrap.com/docs/5.1/content/reboot/

Either way, they're clearly documenting their approach and allows consumers to subscribe or alter that default behavior. Personally -- if I was a consumer of Maker I'd want to be able to specifically target and style the margins of certain tags. One way we could do that with our Theme is having a global default with the ability to customize with a `styles` data option like so: https://theme-ui.com/theme-spec/#styles

We don't have to do any of this right now -- but I thought it worthwhile to queue up some of these approaches so we can dig in when it's appropriate. Reducing the shared styles would be ideal with our Website consumer -- but I just want to be cautious about scoped styles being the only way to get the desired output as that gets incredibly repetitive and error prone to do something as simple as adding a margin on paragraphs.